release/release.js

```diff
@@ -228,7 +231,30 @@ const uploadToS3 = async () => {
 }
 
 const didS3uploadSucceed = async () => {
-  await proceedYesNo('Have all of these commands succeeded?')
+  console.log('Make sure style.css, onfido.min.js and onfido.crossDevice.min.js are in the S3 folder')
```

Comment:
The `S3_FLAGS` env var on lines 225 and 227 should be added to the wiki or to `releaseConfig.js`. I would put it in `releaseConfig.js` as the value of this var does not include confidential information.